go-controller/pkg/ovn/ovn.go

```diff
@@ -609,6 +638,125 @@ func (oc *Controller) GetServiceVIPToName(vip string, protocol kapi.Protocol) (t
 	return namespace, ok
 }
 
+// AddServiceLBToACL associates an empty load balancer with its associated ACL reject rule
+func (oc *Controller) SetServiceACLToLB(lb, vip, acl string) {
+	oc.serviceLBLock.Lock()
+	defer oc.serviceLBLock.Unlock()
+	if _, ok := oc.serviceLBMap[lb]; !ok {
+		oc.serviceLBMap[lb] = make(map[string]*LoadBalancerConf)
+		oc.serviceLBMap[lb][vip] = &LoadBalancerConf{RejectACL: acl}
+		return
+	}
+	if _, ok := oc.serviceLBMap[lb][vip]; !ok {
+		oc.serviceLBMap[lb][vip] = &LoadBalancerConf{RejectACL: acl}
+		return
+	}
+	// This should not happen, the entry should be removed for the VIP when the service is deleted for an existing
```

Comment:
@trozet I don't think this ever can happen, unless we're explicitly setting the map entry to `nil` which the code doesn't do anywhere that I can see. Let's kill the block.

Reply:
@trozet did you decide to keep the block?